collector/filesystem_common.go

```diff
@@ -51,8 +51,11 @@ type filesystemCollector struct {
 	devErrors *prometheus.CounterVec
 }
 
+type filesystemLabels struct {
+	device, mountPoint, fsType string
+}
```

Comment:
Did you run go fmt? I believe that will enforce a newline between type definitions.